File: CS/MyRadioGroup/ExtendedEditor/MyRadioGroupViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.ViewInfo;
using DevExpress.Utils.Drawing;
using DevExpress.XtraEditors.Controls;
using System.Drawing;
using System.Windows.Forms;

namespace EditorDescendant
{
    public class MyRadioGroupViewInfo : RadioGroupViewInfo
    {
        public MyRadioGroupViewInfo(RepositoryItem item)
            : base(item)
        {}



        protected override RadioGroupItemViewInfo CreateItemInfo(DevExpress.XtraEditors.Controls.RadioGroupItem item)
        {
            CheckObjectInfoArgs info = ((CheckObjectInfoArgs)base.CreateItemInfo(item));
            info.AllowHtmlString = ((MyRepositoryItemRadioGroup)OwnerEdit.Properties).AllowHtmlString;

            return (RadioGroupItemViewInfo)info;
        }


        protected override int MinItemHeight
        {
            get
            {
                int heght = ((MyRepositoryItemRadioGroup)OwnerEdit.Properties).MinItemHeight;
                return heght;
            }
        }
    }
}

File: VB/MyRadioGroup/ExtendedEditor/MyRadioGroupViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.ViewInfo
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraEditors.Controls
Imports System.Drawing
Imports System.Windows.Forms

Namespace EditorDescendant
	Public Class MyRadioGroupViewInfo
		Inherits RadioGroupViewInfo
		Public Sub New(ByVal item As RepositoryItem)
			MyBase.New(item)
		End Sub



		Protected Overrides Function CreateItemInfo(ByVal item As DevExpress.XtraEditors.Controls.RadioGroupItem) As RadioGroupItemViewInfo
			Dim info As CheckObjectInfoArgs = (CType(MyBase.CreateItemInfo(item), CheckObjectInfoArgs))
			info.AllowHtmlString = (CType(OwnerEdit.Properties, MyRepositoryItemRadioGroup)).AllowHtmlString

			Return CType(info, RadioGroupItemViewInfo)
		End Function


		Protected Overrides ReadOnly Property MinItemHeight() As Integer
			Get
				Dim heght As Integer = (CType(OwnerEdit.Properties, MyRepositoryItemRadioGroup)).MinItemHeight
				Return heght
			End Get
		End Property
	End Class
End Namespace
